### DISCUSSION TITLE
WinUI 3

### DISCUSSION THREAD

**Discussion by Ireki** [cefsharp/CefSharp#4610] [Ideas]
Are there plans to integrate CefSharp with WinUI 3?

**Comment by amaitland** [comment]
Doesn't `Microsoft` provide a `Chromium` base browser already that works with `WinUI 3`?

**Reply by Ireki** [reply]
Yes, Microsoft provides chromium(WebView 2), which also works with WPF and WinForms. But WebView2 lacks some apis and it's closed source software.

**Reply by amaitland** [reply]
Supporting a new framework is a massive time investment, members from the community would need to volunteer to help with development and maintenance.

[CefSharp.Wpf.HwndHost](https://github.com/cefsharp/CefSharp.Wpf.HwndHost) is already developed outside the main repo, a similar approach could easily be taken. Happy to create a new repo within this organisation if there is enough interest/volunteers.
